java/kotlin-extractor/src/main/kotlin/KotlinFileExtractor.kt

```diff
@@ -3310,26 +3336,31 @@ open class KotlinFileExtractor(
                 extractDefaultsCallArguments(it, e.symbol.owner, callable, enclosingStmt, valueArgs, null, null)
             }
         } else {
-            extractNewExpr(e.symbol.owner, eType.arguments, type, locId, parent, idx, callable, enclosingStmt).also {
```

Comment:
Nitpick: why restyle this so we have to restate `newExprId` at the end, instead of just adding 

```
if (it == null) {
    logger.errorElement("Cannot get newExpr ID", e)
    return
}
```
inside the `also` block?

In general

```
val x = f()
...
x
```

seems like a bit of a Kotlin anti-pattern, and `also` the right tool to replace it.

Reply:
Good point; I'll revert this in a follow-up PR, to avoid blocking the bulk of the change on CI.

Reply:
Ah, that actually doesn't work. Kotlin's type system isn't smart enough to know that the result of `extractNewExpr` is non-null after the `also` block.